ocs_ci/helpers/dr_helpers.py

```diff
@@ -73,3 +73,112 @@ def relocate(preferred_cluster, drpc_name, namespace):
     drpc_obj.wait_for_phase(constants.STATUS_RELOCATED)
 
     config.switch_ctx(prev_index)
+
+
+def get_drpc_name(namespace):
```

Comment:
@prsurve , is there any chance that we might have more than one DRPC ?

Reply:
per namespace we will have only one